### PR TITLE
Adds ask for keeping the changelog title as title

### PR DESCRIPTION
**Pull request by pschoffer**
## Description
Update to the PR template - no need to publish a changelog

### PR template change

Adds an ask to the PR template for keeping the changelog title formatted as a title

### CODE REVIEW

**Review by pauarge — APPROVED** [review]
LGTM

**Comment by pschoffer** [comment]
No deploy needed - no production code change.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ Please don't include internal or private links :)
 
 ## Changelog Description
 <!--
-A description of the context of the change for a changelog. It should have a title, link to the PR, examples (if applicable), and why the change was made.
+A description of the context of the change for a changelog. It should have a title, examples (if applicable), and why the change was made.
+
+**Please keep the changelog title format same as in example bellow (### <Title>), as this is later used to generate the changelog entry title**
```
Comment (GaryJones) [inline .github/PULL_REQUEST_TEMPLATE.md:29]:
```suggestion
**Please keep the changelog title format same as in example below (### <Title>), as this is later used to generate the changelog entry title.**
```

Comment (pschoffer) [reply .github/PULL_REQUEST_TEMPLATE.md:29]: Thanks @GaryJones ! I pushed new PR with the fix - https://github.com/Automattic/vip-go-mu-plugins/pull/2140

**Pull request merged**